vm/src/builtins/singletons.rs

```diff
@@ -52,6 +52,13 @@ impl PyNone {
     }
 }
 
+impl AsNumber for PyNone {
+    const AS_NUMBER: PyNumberMethods = PyNumberMethods {
+        boolean: Some(|number, _vm| Ok(Self::number_downcast(number).bool())),
```

Comment:
```suggestion
        boolean: Some(|number, _vm| Ok(false)),
```
None is singleton. I am pretty sure you can skip actual object value.

Reply:
Good idea. BTW, what is the object's functions such as `repr()` and `bool()` used be for?

Reply:
They are exposed as python method by #[pymethod] macro attached. You can call them by below way.
```python
>>>>> None.__bool__()
False
>>>>> None.__repr__()
'None'
```

Reply:
And those functions @Snowapril referred are used by facade functions `bool(None)` and `repr(None)`